CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ and this project adheres to
 
 ### Added
 
+- [#4428](https://github.com/firecracker-microvm/firecracker/pull/4428): Added
+  basic ACPI support to Firecracker for x86_64 microVMs. This allows booting
```

Comment:
`basic ACPI support` creates more questions. Instead it would be better to just say something like "Added ACPI tables to pass information of available vCPUs, interrupt controllers, etc.".

Reply:
Reworded as suggested